ods_ci/tests/Tests/0400__ods_dashboard/0401__ods_dashboard.robot

```diff
@@ -278,19 +248,17 @@ Verify Error Message In Logs When All Authenticated Users Are Set As RHODS Admin
 Verify Dashboard Pod Is Not Getting Restarted
     [Documentation]    Verify Dashboard Pod container doesn't restarted
     [Tags]    Sanity
-    ...       Tier1
     ...       ODS-374
     ${pod_names}    Get POD Names    ${APPLICATIONS_NAMESPACE}    app=${DASHBOARD_APP_NAME}
     Verify Containers Have Zero Restarts    ${pod_names}    ${APPLICATIONS_NAMESPACE}
 
 Verify Switcher to Masterhead
     [Tags]    ODS-771
-    ...       Tier2
+    ...       Smoke
```

Comment:
Smoke?

Reply:
yes, I changed tags according to importance. This one is important to be in Smoke.